app/src/main/java/de/xikolo/models/Quiz.java

```diff
@@ -47,7 +45,6 @@ public Quiz convertToRealmObject() {
             quiz.instructions = instructions;
             quiz.timeLimit = timeLimit;
             quiz.allowedAttempts = allowedAttempts;
-            quiz.maxPoints = maxPoints;
```

Comment:
You forgot to remove lines 35 and 36 in the `Quiz.JsonModel`

```
@Json(name = "max_points")
public int maxPoints;
```

Reply:
Good catch! Fixed in https://github.com/openHPI/xikolo-android/commit/2b3b5e6420a5e1dbbce44293bb9e5038201a2d89.